pkg/cloudfabric-controller/deployment/deployment_controller.go

```diff
@@ -155,7 +156,7 @@ func NewDeploymentController(dInformer appsinformers.DeploymentInformer, rsInfor
 }
 
 // Run begins watching and syncing.
-func (dc *DeploymentController) Run(workers int, stopCh <-chan struct{}) {
+func (dc *DeploymentController) Run(controllerConfig config.ControllerConfig, stopCh <-chan struct{}) {
```

Comment:
Run() function has single signature as the original. Try to keep it the same. We are trying to make the migration of controllers from Kube CM to workload CM as easy as possible